descriptors/ModuleDescriptor-template.json

```diff
@@ -195,6 +195,17 @@
         }
       ]
     },
+    {
+      "id": "orders-storage.order-lines",
```

Comment:
Do we really need separate interface for this endpoint or it can be added to `orders-storage.po-lines`?

`GET /orders-storage/orders` is part of `orders-storage.purchase-orders`. Depending on the selected approach, new interface might have to be created for this endpoint as well to be consistent with order lines endpoint.

Reply:
I created a new interface just because we will be able to query purchase orders too not just po lines. I am ok to move it to po-lines because we are returning just po-lines

Reply:
Done